openrndr-core/src/main/kotlin/org/openrndr/shape/ShapeContour.kt

```diff
@@ -301,6 +321,13 @@ data class ShapeContour(val segments: List<Segment>, val closed: Boolean, val po
         }
     }
 
+    /**
+     * Calculates pose [Matrix44] that describes an orthonormal basis
+     * formed by normal and tangent of the contour at [t](https://pomax.github.io/bezierinfo/#explanation).
+     *
+     * Which means it returns a [Matrix44],
```

Comment:
this explanation is not included in a another `pose` function

Reply:
Could also mention that the matrix includes position and rotation as I mentioned in the other pose